sdk/test-utils/recorder-new/recordings/browsers/proxy_tool/recording_sample_response.json

```diff
@@ -1,39 +0,0 @@
-{
-  "Entries": [
```

Comment:
why delete them?
let's keep them to show how the recordings look like?

Reply:
The directory is being ignored now (per the reason above, it creates a diff every test run otherwise). If we keep them checked in they will become outdated as we make changes to the tests, which could get confusing.

Reply:
I was thinking, we'll keep one recording always for representational purposes.
Whenever there is a significant change, we'll override the `.gitignore` and check them in anyways.
What do you think?

Reply:
Feels like overriding when there's a significant change is something that we will easily forget to do... if people do want to see the recordings, there are a couple of options:
1. They can look at another project like the template package (IMO this is a more sensible place for examples to be anyway)
2. If they really want to see the recordings in this particular package, they can run the tests themselves and get fresh ones

IMHO it's not worth keeping them around.

Reply:
okay, fine with deleting